File: 1051_Height_Checker.js
var heightChecker = function(heights) {
    let arr = [...heights];
    arr = arr.sort((a,b) => a-b);
    let i = 0;
    let count = 0;
    while(i < heights.length){
        if(heights[i] != arr[i]) count++;
        i++;
    }
    return count;
};

File: 1051_Height_Checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        int n = heights.size();
        vector<int> arr(n);
        for(int i = 0; i < n; i++)
            arr[i] = heights[i];
        int ans = 0;
        sort(arr.begin(), arr.end());
        for(int i = 0; i < n; i++)
            if(arr[i] != heights[i]) ans++;
        return ans;
    }
};
